qtconsole/console_widget.py

```diff
@@ -223,6 +223,9 @@ def __init__(self, parent=None, **kw):
             self.setParent(parent)
 
         self._is_complete_msg_id = None
+        self._is_complete_timeout = 0.25
```

Comment:
Could this time be reduced a bit more? Say 0.1?

Reply:
0.1 works as well.